### PR TITLE
Fixed all the components

### PR DESCRIPTION
**Pull request by adars-kafle**

### CODE REVIEW

File: src/components/TaskComponents/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import AllTasks from "./AllTasks";
```
Comment (sudip-sth) [inline src/components/TaskComponents/index.ts]: No need to import first. 
`export * from "./FIlename"`

Comment (adars-kafle) [reply src/components/TaskComponents/index.ts:1]: Oh yes, thank you for the feedback

File: src/context/TaskContext.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  const queryClient = useQueryClient();
+
+  const {
+    data: tasks = [],
+    isLoading,
+    isError,
+  } = useQuery<Task[]>("tasks", taskService.getTasks);
+
+  const addTaskMutation = useMutation(taskService.addTask, {
+    onSuccess: () => {
```
Comment (sudip-sth) [inline src/context/TaskContext.tsx:23]: Add queries and mutation in context if its needed to be used in multiple components in different heirarchy

File: src/components/TaskComponents/index.ts (hunk 1 above)
Comment (adars-kafle) [reply src/components/TaskComponents/index.ts]: I resolved this as you said.

**Pull request merged**